tools/cdk/app.ts

```diff
@@ -1,13 +1,17 @@
 #!/usr/bin/env node
 import "dotenv/config";
 
-import { App, StackProps } from "aws-cdk-lib";
+import { type StackProps, App } from "aws-cdk-lib";
```

Comment:
```suggestion
import { App } from "aws-cdk-lib";
import type { StackProps } from "aws-cdk-lib";
```
Two reasons that I prefer this (that are mostly trivial)
1. Vim doesn't syntax highlight the mixed line properly for me lol
2. There's actually a prescribed import order by ESLint, and it's easier for me to fulfill them when I just split them up. https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md